cnd/src/network.rs

```diff
@@ -238,9 +238,7 @@ pub struct ComitNode {
     #[behaviour(ignore)]
     bitcoin_addresses: HashMap<identity::Bitcoin, crate::bitcoin::Address>,
     #[behaviour(ignore)]
-    order_swap_ids: HashMap<OrderId, LocalSwapId>,
-    #[behaviour(ignore)]
-    confirmed_order_peers: HashMap<OrderId, PeerId>,
+    order_to_swap_id: HashMap<OrderId, LocalSwapId>,
```

Comment:
Uniformity in the code is really important to increase maintainability and reduce cognitive burden on future readers of the code. In this instance all the other hash maps use plural, please either be uniform or do a PR that changes all of them (with justification).